pupil-spa/src/app/storage.service.spec.ts

```diff
@@ -1,12 +1,12 @@
 import { TestBed, inject } from '@angular/core/testing';
-import { StorageService } from './storage.service';
+import { StorageService, StorageKey } from './storage.service';
 
 let service: StorageService;
 
 describe('StorageService', () => {
 
   const shouldNotExecute = () => {
-    expect(1).toBe(2);
+    expect('this code not').toBe('executed');
```

Comment:
If you are passing `done()` into the test you can exit the test with an error by calling
`done(new Error('Should not have been called'))`